templates/shared/forms/interactive/managed-observability.html

```diff
@@ -0,0 +1,143 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description" >
+    <header class="p-modal__header" style="display: block; border-bottom: 0">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem" > Close </button>
+      <h2 class="p-modal__title u-sv1" id="modal-title"> You innovate, we run the monitoring tools! </h2>
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Which monitoring tools do you want to use?</h3>
+        <div class="row u-no-padding u-equal-height">
+          <div class="col-6 u-sv3">
+            <input type="checkbox" id="tools-prometheus" />
+            <label for="tools-prometheus">Prometheus</label>
+            <input type="checkbox" id="tools-cortex" />
+            <label for="tools-cortex">Cortex</label>
+            <input type="checkbox" id="tools-grafana" />
+            <label for="tools-grafana"> Grafana</label>
+            <input type="checkbox" id="tools-grafana-loki" />
+            <label for="tools-grafana-loki"> Grafana Loki</label>
+            <input type="checkbox" id="tools-jaeger" />
+            <label for="tools-jaeger"> Jaeger</label>
+          </div>
+          <div class="col-6 u-sv3">
+            <input type="checkbox" id="tools-elasticsearch" />
+            <label for="tools-elasticsearch"> Elasticsearch</label>
+            <input type="checkbox" id="tools-graylog" />
+            <label for="tools-graylog">Graylog</label>
+            <input type="checkbox" id="tools-influxdb" />
+            <label for="tools-influxdb">InfluxDB</label>
+            <input type="checkbox" id="tools-other" />
+            <label for="tools-other"> Other</label>
```

Comment:
For the other check box, you can use something like [this](https://github.com/canonical-web-and-design/ubuntu.com/blob/main/templates/shared/forms/interactive/openstack.html#L52) to allow a custom input.